ref/net46/Microsoft.Build/Microsoft.Build.cs

```diff
@@ -1402,6 +1425,25 @@ public partial class DistributedFileLogger : Microsoft.Build.Framework.IForwardi
         public void Initialize(Microsoft.Build.Framework.IEventSource eventSource, int nodeCount) { }
         public void Shutdown() { }
     }
+    public partial class EventArgsDispatcher : Microsoft.Build.Framework.IEventSource
```

Comment:
There's no need as such, I was just following the pattern with the other loggers, and I thought exposing the reader and writer might be useful for other purposes.